js/samples/advanced-nodejs/views/document.html

```diff
@@ -57,25 +59,22 @@ <h1 id='ir-title'>Geography</h1>
     </main>
 
     <script type='text/javascript'>
-        async function handleLaunchImmersiveReader() {
-            const data = {
+        function handleLaunchImmersiveReader() {
+            var data = {
                 title: $('#ir-title').text().trim(),
                 chunks: [{
                     content: $('#ir-text').text().trim(),
                     lang: 'en'
                 }]
             };
-            const token = await getImmersiveReaderTokenAsync();
-            const subdomain = await getSubdomainAsync();
-
-            const options = {
+            
+            var options = {
                 'onExit': exitCallback
             };
 
-            ImmersiveReader.launchAsync(token, subdomain, data, options)
-                .catch ((error) => {
-                    console.log('there was an error', error);
-                });
+            // Refer to the launchImmersiveReader() function in helpers.js for more information. It covers things like retrieving
```

Comment:
nit: change "covers things like retrieving" to "covers how to retrieve"?

Reply:
Tweaked the comments.